src/stubs.ts

```diff
@@ -22,6 +23,12 @@ interface StubOptions {
   renderStubDefaultSlot?: boolean
 }
 
+const stubsMap: WeakMap<ComponentOptions, VNodeTypes> = new WeakMap()
```

Comment:
I wonder if we should export this and do a `stubsMap.clear()` each time we mount a component? Although `WeakMap` does save the effort of cleaning up, it might be more reliable to additionally blow it away ourselves. What do you think?

Reply:
I've also thought about that, but would like to avoid this. If we have two wrappers mounted in parallel, and we interact with them, since stubbing occurs on each VNode tree update clearing will result in unexpected behaviour, since `stubsMap` is shared across all instances of "apps" created by `mount`

I definitely do not want (at least for now) making some "context" for separate trees and pushing it elsewhere and having "cleanup" happening in forced & uncontrolled manner (uncontrolled because we don't know when next `mount` is called) makes me believe I'm introducing more complexity than I should

Reply:
Alright lets go with this. If we notice any weirdness in the future, we can look into it.